static/js/admin.js

```diff
@@ -894,7 +906,7 @@ function getStudents(sectionName) {
     obj.onreadystatechange = function () {
 
         if (obj.readyState == 4 && obj.status == 200) {
-            students = JSON.parse(obj.responseText);
+            var students = JSON.parse(obj.responseText);
```

Comment:
Where are all of the uses of students above getting their value, especially if this is declared local, and you have removed all the other calls to JSON.parse.